### ISSUE TITLE
Vatras Message Spam: npc_getstatetime/npc_setstatetime

### ISSUE THREAD

**Issue by markusobi** [REGoth-project/REGoth#113]
As far as I understand the reason behind his message spam is, that `npc_getstatetime` and `npc_setstatetime` are not implemented/registered yet. [npc_setstatetime](https://github.com/REGoth-project/REGoth/blob/master/src/logic/scriptExternals/Stubs.cpp#L1914) is registered but does nothing. 
The Daedalus script function triggering his speaches is the following:
```
func int ZS_Preach_Vatras_loop()
{
	if (Npc_GetStateTime(self) >= 13)
	{
		if (self.aivar[AIV_TAPOSITION] > 20)
		{
			self.aivar[AIV_TAPOSITION] = 0;
		};
	
		B_Preach_Vatras(self.aivar[AIV_TAPOSITION]);
		
		self.aivar[AIV_TAPOSITION] = self.aivar[AIV_TAPOSITION] + 1;		
		
		Npc_SetStateTime(self, 0);
	};
	
	return LOOP_CONTINUE;
};
```

**Comment by degenerated1123** [comment]
There should be a field for the statetime in the NPCs state-machine. Just move the stubs to externals.cpp and write a getter/setter for that.

**Comment by markusobi** [comment]
fixed by #180
